apps/desktop/desktop/src/main/java/bisq/desktop/util/TextFlowUtils.java

```diff
@@ -0,0 +1,47 @@
+package bisq.desktop.util;
```

Comment:
I think the`bisq.desktop.components.controls` package would be a better place. Maybe a util subpackage there. We have also `bisq.desktop.common.utils` but that is more for not UI controls specific stuff.

Reply:
I agree, I couldn't find a better place when I was doing it I'll refactor it now